llvm/test/tools/sycl-post-link/device-code-split/per-aspect-split-1.ll

```diff
@@ -10,30 +10,30 @@
 ; RUN: sycl-post-link -split=auto -symbols -S %s -o %t.table
 ; RUN: FileCheck %s -input-file=%t_0.ll --check-prefixes CHECK-M0-IR \
 ; RUN: --implicit-check-not kernel0 --implicit-check-not kernel1
-; RUN: FileCheck %s -input-file=%t_1.ll --check-prefixes CHECK-M1-IR \
-; RUN: --implicit-check-not kernel0 --implicit-check-not kernel1
-; RUN: FileCheck %s -input-file=%t_2.ll --check-prefixes CHECK-M2-IR \
-; RUN: --implicit-check-not kernel0 --implicit-check-not kernel1
+; RUN: FileCheck %s -input-file=%t_1.ll --check-prefixes CHECK-M2-IR \
```

Comment:
It is ok to change this test, per-reqd-wg-size-split-2.ll, sycl-esimd-large-grf.ll and sycl-large-grf.ll below: by adding new optional kernel feature reqd-sub-group-size to internal data structures, re-hashing happened which affected name generation logic. This does not affect customers as customers don't use these temp files directly, and didn't break anything in the pipeline.